.github/workflows/continuous-integration.yaml

```diff
@@ -16,7 +16,8 @@ jobs:
     strategy:
       matrix:
         node-version: [10, 12, 14]
-        mongodb-version: [4.4] # the latest stable version
+        mongodb-version: [3.6, 4.4, 5.0]
```

Comment:
according to https://www.mongodb.com/support-policy/lifecycles, mongodb v3.6 has reached EOL last April. Do we still want to include that?

Reply:
Since connector 5.x was released when MongoDB 3.x was still supported, I wanted to make sure it still works with that version (i.e. Dropping a MongoDB version should require a semver major on the connector as well).